File: AD-HOC/3060/main.c
#include <stdio.h>

void printValues(int v, int p) {
    int value = v / p;
    int adjustment = v % p;
    for (int i = 0; i < p; i++)
        printf("%d\n", i < adjustment ? value + 1 : value);
}

int main(int argc, char const *argv[]) {
    int v, p;
    scanf("%d %d", &v, &p);
    printValues(v, p);
    return 0;
}

File: AD-HOC/3060/main.rb
def print_values(v, p)
  value = v / p
  adjustment = v % p
  p.times do |i|
    puts (i < adjustment) ? value + 1 : value
  end
end

v = gets.to_i
p = gets.to_i
print_values(v, p)
